n_i2c.c

```diff
@@ -112,11 +114,36 @@ const char *i2cNoteTransaction(char *request, char **response)
 
     delayIO();
 
+    // `web.*` transactions are unpredictable because we do not know the
+    // associated payload size or the network conditions. The default timeout
+    // for a `web.*` transaction is 90-seconds. Therefore a perfect functioning
+    // transaction could exceed `NOTECARD_INTER_TRANSACTION_TIMEOUT_SEC`, and
+    // an error would be returned to the caller when no error condition exists.
+    // By interrogating the request, we can discover if the request is a
+    //`web.*` transaction. If so, we will update the timeout to the default
+    // 90-seconds, or the value of the `seconds` parameter specified by the
+    // caller (who better understands context of the `web.*` transaction).
+    size_t transaction_timeout_seconds = NOTECARD_INTER_TRANSACTION_TIMEOUT_SEC;

```

Comment:
Just an idea, but could we move this logic up to the level of `_Transaction` so that we don't have to repeat it for both serial and I2C? You would also have access to the `J *` request rather than having to deal with raw strings, which would make it easier to pull out the `seconds` parameter, if it exists. You'd then pass down the transaction timeout seconds as a new parameter to `i2cNoteTransaction`/`serialNoteTransaction`.

Reply:
Also, mixing snake_case and camelCase.

Reply:
death to camels!

Reply:
Camels stomp snakes. :frowning_face: